client/src/pages/Feed.js

```diff
@@ -258,20 +261,57 @@ const Feed = () => {
     dispatchAction(TOGGLE_STATE, "showFilters");
   };
 
-  useEffect(() => {
+  const loadPosts = useCallback(async () => {
+    const limit = 5;
```

Comment:
Should we make this a constant? There was also a discussion in one of the backend PRs for the Feed about creating a central file to store constants so that the backend and frontend are on the same page about data properties.  I'm not sure if that has been created yet.

Reply:
Reviewed and this file doesn't exist yet.